src/Orleans.Streaming/SimpleMessageStream/SimpleMessageStreamProducerExtension.cs

```diff
@@ -184,10 +183,10 @@ internal void AddRemoteSubscriber(GuidId subscriptionId, IStreamConsumerExtensio
             internal void RemoveRemoteSubscriber(GuidId subscriptionId)
             {
                 consumers.TryRemove(subscriptionId, out _);
-                if (consumers.Count == 0)
-                {
-                    // Unsubscribe from PubSub?
-                }
+                //if (consumers.IsEmpty)
+                //{
+                //    // Unsubscribe from PubSub?
+                //}
```

Comment:
Prefer we delete this code instead, potentially leaving the original comment the note for future spelunkers